pyaedt/icepak.py

```diff
@@ -3834,11 +3844,11 @@ def assign_solid_block(
         """
         if not self.modeler.get_object_from_name(object_name).solve_inside:
             self.logger.add_error_message(
-                "Please use ``assign_hollow_block`` function with this object as" "``solve_inside`` is ``False``."
+                "Use ``assign_hollow_block`` function with this object as" "``solve_inside`` is ``False``."
             )
             return None
         if ext_temperature != "AmbientTemp" and ext_temperature is not None and not htc:
-            self.logger.add_error_message("Please set an argument for ``htc`` or remove ``ext_temperature`` argument.")
+            self.logger.add_error_message("Set an argument for ``htc`` or remove ``ext_temperature`` argument.")
```

Comment:
```suggestion
            self.logger.add_error_message("Set an argument for ``htc`` or remove the ``ext_temperature`` argument.")
```
Again--if you don't want to change the messages (and update existing tests) to add articles like "the", I do understand.